creatorkit/Project 1.md

```diff
@@ -0,0 +1,40 @@
+![Creator Logo](../creatorlogo.png)
+
+# Project 1 - IoT Button
+
+## Introduction
+
+This project will allow you to press a button on one of your Clicker boards and see an LED toggle on the other Clicker board. Ci40 acts as the hub for the 6LoWPAN communication. You can also view the inter-device messages via the FlowM2M website - demonstrating connectivity with the cloud.
+
+This project requires the Ci40 and 2 Clicker boards from Creator Kit. It does not require any of the 3 Click expansion boards.
+
+## Steps
+
+To complete these steps, refer to the [Toolbox](Toolbox.md) for detailed descriptions of each action. You will need a [FlowM2M account](Toolbox.md#creating-a-flowm2m-account) too.
+
+* Get the code: [Pre-built Release](http://github.com/creatorkit) or [Source Code](http://github.com/creatorkit)
+* [Flash](Toolbox.md#programming-a-6lowpan-clicker) the ""lwm2m-client-contiki-button"" hex file onto one Clicker board - Make a note of which one it is
+* [Flash](Toolbox.md#programming-a-6lowpan-clicker) the ""lwm2m-client-contiki-led"" hex file onto the second Clicker board - Make a note of which one it is
+* [Connect](Toolbox.md#connecting-ci40-to-the-internet) Ci40 to the Internet
+* On your Ci40, run the ""command"" command in the [serial terminal](Toolbox.md#connecting-to-ci40-via-serial)
+* [Provision](Toolbox.md#provisioning-ci40) Ci40
```

Comment:
Link is not working correctly. i think you need to have different labels for both provisioning.